quests/rl/hyperparam.yaml

```diff
@@ -2,7 +2,7 @@
 trainingInput:
   scaleTier: BASIC_GPU
   hyperparameters:
-    maxTrials: 50
+    maxTrials: 30
     maxParallelTrials: 30
```

Comment:
This does not take advatange of Bayesian Optimization if we set MaxParallelTrials = maxTrials. Perhaps reduce it to a lower number(with the tradeoff of completing faster)

Reply:
Yeah, I was really heartbroken about that too. Did a lot of experimentation, and this was statistically the best number to set it at to get the agent to reach a +100 score within 20 minutes.

Reply:
Ok, makes sense, maybe in the write up can mention this as a footnote.